### DISCUSSION TITLE
Breakout Room 8- Practical Series4

### DISCUSSION THREAD

**Discussion by BakariMtili** [IBAHCM/RPiR#270] [Archive]
Hello here are the people and their usernames
Bakari Mtili @BakariMtili
Lucy Parker @lucyparker25
Emanuel Hape @EmmanuelHape

**Comment by AilsaWatson** [comment]
Hey guys this is the link to my series 04 repo - https://github.com/IBAHCM/AilsaWatsonBCI.git

I'll post a new issue about its status atm and grant access to you all

**Comment by BakariMtili** [comment]
Here are our repos:
https://github.com/IBAHCM/EmmanuelHapeBCI.git
https://github.com/IBAHCM/lucyparker25BCI.git
https://github.com/IBAHCM/BakariMtiliBCI.git
 https://github.com/IBAHCM/AilsaWatsonBCI.git

**Reply by BakariMtili** [reply]
Therefore here is the updated list of people and their usernames
Bakari Mtili @BakariMtili
Lucy Parker @lucyparker25
Emanuel Hape @EmmanuelHape
Aisla Watson @AilsaWatson

**Comment by AilsaWatson** [comment]
Cheers Bakari,

And just to let everyone know here I have posted the link for my repo on the groups discussion in registry, and have created a new issue for 04.

Sent from Mail<https://go.microsoft.com/fwlink/?LinkId=550986> for Windows 10
